src/main/java/io/swagger/service/MovieUserDetailsService.java

```diff
@@ -37,4 +38,20 @@ public UserDetails loadUserByUsername(String email) throws UsernameNotFoundExcep
                 user.get().getPassword(),
                 Collections.singletonList(new SimpleGrantedAuthority(role)));
     }
+
+    public UserDetails loadUserById(UUID id) {
```

Comment:
If the security context can have the UserId stored in addition to the username and password, then this call is not necessary. If the function above, `loadUserByUsername`, can stuff the UserId, then that will make the call to check the OwningUserId much easier.